flag_engine/features/models.py

```diff
@@ -52,7 +54,9 @@ def get_feature_state_value(self):
         return self.get_value()
 
     def _get_multivariate_value(self, identity_id: int) -> typing.Any:
-        percentage_value = get_hashed_percentage_for_object_ids([self.id, identity_id])
+        percentage_value = get_hashed_percentage_for_object_ids(
+            [self.django_id or self.featurestate_uuid, identity_id]
```

Comment:
Should we write a test / parametrize an existing test to confirm that `django_id` and `featurestate_uuid` both work?

Reply:
Ah okay